Description: Rename fio command to fiona to avoid name clash
 There is already another package providing a binary "fio" (fio).
Author: Johan Van de Wauw <johan.vandewauw@gmail.com>
Forwarded: not-needed

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -57,7 +57,7 @@ test = [
 ]
 
 [project.scripts]
-fio = "fiona.fio.main:main_group"
+fiona = "fiona.fio.main:main_group"
 
 [project.entry-points."fiona.fio_commands"]
 bounds = "fiona.fio.bounds:bounds"
